modules/ztp-pre-staging-booting-from-live-os.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-pre-staging-tool.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-booting-from-live-os_{context}"]
+= Booting from a live OS
+
+You can use the {factory-prestaging-tool} with servers where only one disk is available and external disk drive cannot be attached to the server.
+
+// I'm wondering if the user needs to know this background on line 12-13.
+As part of the ZTP workflow, the Assisted Installer (AI) uses the `coreos-installer` utility to write {op-system-first} to the disk.
+Therefore, if you boot from a pre-installed {op-system} on a single disk, the utility throws an error because the device is in use and cannot finish the writing process.
+
+[WARNING]
+====
+{op-system} requires the disk to not be in use when the disk is about to be written by an {op-system} image.
+====
+
+.Prerequisites
+
+* You have installed `podman`
+
+.Procedure
+
+. To install the images on a Dell Server, run the following script:
++
+[source,terminal]
+----
+#/bin/bash
+
+CONTAINER_TOOL=podman
+IDRAC_IP=$1
+
+${CONTAINER_TOOL} run --network host --rm -it quay.io/alosadag/racadm:latest -r $IDRAC_IP -u root -p $PASSWD- remoteimage -d
+${CONTAINER_TOOL} run --network host --rm -it quay.io/alosadag/racadm:latest -r $IDRAC_IP -u root -p $PASSWD- remoteimage -c -l http://10.19.138.94/rhcos-4.10.16-x86_64-live.x86_64.iso
```

Comment:
correct, you need to include the location of the RHCOS live ISO. Usually it is local since it is faster and works in disconnected env, where it is common in telco.

Reply:
Also, we definitely can't link to non-RH vetted images like `quay.io/alosadag/racadm:latest`. It has to be officially hosted.